src/extensions/export/bootstrap-table-export.js

```diff
@@ -127,6 +128,7 @@ $.BootstrapTable = class extends $.BootstrapTable {
 
     let $menu = $(this.constants.html.toolbarDropdown.join(''))
     let $items = this.$export
+
```

Comment:
Do we really need that useless new lines after each variable ?
In my opion the code its worser to read.

Reply:
I think it will be more readable and less crowded.

https://eslint.org/docs/rules/padding-line-between-statements

![image](https://user-images.githubusercontent.com/2117018/93674353-2a71b480-fade-11ea-9e77-c1a5a7b294a1.png)

Reply:
If this is a recommendation from eslint, then lets do it :+1: